route-rs-runtime/src/link/mod.rs

```diff
@@ -1,14 +1,29 @@
+//! # What are they for?
+//!
+//! Links are an abstraction used by the runtime to link processors and classifiers together, and manage the flow of packets through the router.
+//! Processors and Classifiers, which implement all the non-flow business logic of the router, are loaded into links to create specfic behavior.
+//! Links are joined together via their interfaces, and the links are then dumped into a runtime to begin pulling packets through the router.
```

Comment:
> links are then dumped into a runtime to begin pulling packets...

We can do a better job of explaining what the runtime is for here. How about something like:
> Links are composed together via the `Link` trait, and handed to Tokio (some Links, such as `QueueLink` are async and rely on Tokio to pull packets from upstream).